dashboard/app/models/user.rb

```diff
@@ -90,6 +90,7 @@
 class User < ApplicationRecord
   include SerializedProperties
   include SchoolInfoDeduplicator
+  include LevelProgressable
   include LocaleHelper
   include UserMultiAuthHelper
   include UserPermissionGrantee
```

Comment:
Any tips on the right order of inclusion?
Should we follow alphabetical order or use a different approach?

Reply:
I wouldn't think ordering mattered on the `include` unless one concern has a dependency on another, which would be an anti-pattern. By the time the methods in these concerns are invoked later in the code, all of these concerns should have already been loaded.